addons/website/static/src/snippets/s_website_form/options.js

```diff
@@ -1426,7 +1437,7 @@ options.registry.WebsiteFieldEditor = FieldEditor.extend({
         // Update available visibility dependencies
         const selectDependencyEl = uiFragment.querySelector('we-select[data-name="hidden_condition_opt"]');
         const existingDependencyNames = [];
-        for (const el of this.formEl.querySelectorAll('.s_website_form_field:not(.s_website_form_dnone)')) {
+        for (const el of this.formEl.querySelectorAll('.s_website_form_field:not(.s_website_form_dnone), .s_website_form_field[data-type]')) {
```

Comment:
Same here, since you modify the line, you can linter it with the double quotes. Check the rest of your diff. :+1: